docs/deployment/k8s/index.mdx

```diff
@@ -96,48 +120,126 @@ spec:
   issuers:
     # This needs to be the public key of a cluster seed generated by
     # `wash keys gen cluster`
-    - CDK...
+    - <public-key>
   # The lattice to connect the hosts to
   lattice: 83a5b52e-17cf-4080-bac8-f844099f142e
   # Additional labels to apply to the host other than the defaults set in the operator
   hostLabels:
     some-label: value
+  # The address to connect to nats 
+  natsAddress: nats://nats-cluster.default.svc.cluster.local
   # Which wasmCloud version to use
-  version: 0.82.0
+  version: 1.0.2
   # The name of a secret in the same namespace that provides the required secrets.
-  secretName: cluster-secrets
+  secretName: my-wasmcloud-cluster
   # Enable the following to run the wasmCloud hosts as a DaemonSet
   #daemonset: true
   # The name of the image pull secret to use with wasmCloud hosts so that they
   # can authenticate to a private registry to pull components.
   # registryCredentialsSecret: my-registry-secret
 ```
+```shell
+kubectl apply -f config.yaml
+```
 
-The CRD requires a Kubernetes Secret with the following keys:
+Next we will create a manifest called `secret.yaml` for our secret, adding the seed generated above in the `WASMCLOUD_CLUSTER_SEED` field:
 
 ```yaml
 apiVersion: v1
 kind: Secret
 metadata:
   name: my-wasmcloud-cluster
-data:
+stringData:
   # You can generate this with wash:
   # `wash keys gen cluster`
-  # For more on wash see https://wasmcloud.com/docs/installation
   WASMCLOUD_CLUSTER_SEED: <seed>
+#data:

```

Comment:
I would probably just remove the `#data:` and the lines following it entirely, I don't know it adds anything in to the example